Source/Core/Core/AchievementManager.h

```diff
@@ -121,6 +121,9 @@ class AchievementManager
     int failed_login_code = 0;
   };
   using UpdateEvent = Common::HookableEvent<"AchievementManagerUpdate", const UpdatedItems&>;
+#ifdef ANDROID
+  using LoginEvent = Common::HookableEvent<"AchievementManagerLogin", const int&>;
+#endif  // ANDROID
```

Comment:
There's no need to ifdef this (and the corresponding code in the .cpp file). Having the event there for non-Android builds even though there's currently no code using it doesn't do any harm.